child_image/multiprotocol_rpmsg/prj_no_dfu.conf

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
```

Comment:
prj_no_dfu.conf and prj_release.conf files are not needed in the mcuboot and multiprotocol_rpmsg directories, as long, as we have only one prj.conf file in the main sample directory.